src/articles/ArticlePreview.js

```diff
@@ -107,47 +111,64 @@ export default function ArticlePreview({
 
   return (
     <s.ArticlePreview>
-      <SmallSaveArticleButton
-        api={api}
-        article={article}
-        isArticleSaved={isArticleSaved}
-        setIsArticleSaved={setIsArticleSaved}
-      />
-
-      <s.Title>{titleLink(article)}</s.Title>
-      <ArticleSourceInfo
-        articleInfo={article}
-        dontShowPublishingTime={dontShowPublishingTime}
-        dontShowSourceIcon={dontShowSourceIcon}
-      ></ArticleSourceInfo>
-      <s.ArticleContent>
-        {article.img_url && <img alt="" src={article.img_url} />}
-        <s.Summary>{article.summary}...</s.Summary>
-      </s.ArticleContent>
-
-      <s.BottomContainer>
-        <s.Topics>
-          {topics.map((topic) => (
-            <span key={topic}>{topic}</span>
-          ))}
-        </s.Topics>
-        <ArticleStatInfo
-          cefr_level={cefr_level}
-          articleInfo={article}
-        ></ArticleStatInfo>
-      </s.BottomContainer>
-      {article.video ? (
-        <img
-          alt=""
-          style={{ float: "left", marginRight: "1em" }}
-          src={
-            "https://img.youtube.com/vi/" +
-            extractVideoIDFromURL(article.url) +
-            "/default.jpg"
-          }
+      {!isUnfinishedArticle && (
+        <SmallSaveArticleButton
+          api={api}
+          article={article}
+          isArticleSaved={isArticleSaved}
+          setIsArticleSaved={setIsArticleSaved}
         />
-      ) : (
-        ""
+      )}
+
+      {isUnfinishedArticle && (
+        <div>
+          <s.UnfinishedArticleContainer>
+            <s.Title>{titleLink(article)}</s.Title>
+            {article.img_url && <img alt="" src={article.img_url} />}
+          </s.UnfinishedArticleContainer>
+          <div style={{ fontWeight: "550" }}>
+            ({article.time_until_last_read},{" "}
+            {Math.round(article.last_reading_percentage * 100)}% read)
+          </div>
+        </div>
+      )}
+      {!isUnfinishedArticle && (
```

Comment:
Seems to me that the `ArticlePreview` and the `UnfinishedArticlePreview` are rendered exclusive from each other, so they might actually have been two separate components anyway?

Reply:
I have created a new component for the UnfishiedArticles